types/node/http.d.ts

```diff
@@ -113,7 +113,86 @@ declare module "http" {
     }
     // outgoing headers allows numbers (as they are converted internally to strings)
     type OutgoingHttpHeader = number | string | string[];
-    interface OutgoingHttpHeaders extends NodeJS.Dict<OutgoingHttpHeader> {}
+    interface OutgoingHttpHeaders extends NodeJS.Dict<OutgoingHttpHeader> {
+        accept?: string | undefined;
+        'accept-charset'?: string | undefined;
+        'accept-encoding'?: string | undefined;
+        'accept-language'?: string | undefined;
+        'accept-ranges'?: string | undefined;
+        'access-control-allow-credentials'?: string | undefined;
+        'access-control-allow-headers'?: string | undefined;
+        'access-control-allow-methods'?: string | undefined;
+        'access-control-allow-origin'?: string | undefined;
+        'access-control-expose-headers'?: string | undefined;
+        'access-control-max-age'?: string | undefined;
+        'access-control-request-headers'?: string | undefined;
+        'access-control-request-method'?: string | undefined;
+        age?: string | undefined;
+        allow?: string | undefined;
+        authorization?: string | undefined;
+        'cache-control'?: string | undefined;
+        'cdn-cache-control'?: string | undefined;
+        connection?: string | undefined;
+        'content-disposition'?: string | undefined;
+        'content-encoding'?: string | undefined;
+        'content-language'?: string | undefined;
+        'content-length'?: string | number | undefined;
```

Comment:
This assumption is dangerously wrong. headers are usually strings and have to be casted manually.

Reply:
In examples and tests Node.js team often use number for 'content-length'. The same in real world. Not sure that need to break it.

Reply:
Do you have a link?

Reply:
Ok, it will converted to string internally...